src/pages/individualGallery/IndividualGallery.jsx

```diff
@@ -0,0 +1,166 @@
+import React from 'react';
+import { useIntl } from 'react-intl';
+import { useParams } from 'react-router-dom';
+import { capitalize, transform } from 'lodash-es';
+
+import { useTheme } from '@material-ui/core/styles';
+
+import useDocumentTitle from '../../hooks/useDocumentTitle';
+import useIndividual from '../../models/individual/useIndividual';
+import { deriveIndividualName } from '../../utils/nameUtils';
+import errorTypes from '../../constants/errorTypes';
+import CustomAlert from '../../components/Alert';
+import Button from '../../components/Button';
+import Link from '../../components/Link';
+import LoadingScreen from '../../components/LoadingScreen';
+import MainColumn from '../../components/MainColumn';
+import SadScreen from '../../components/SadScreen';
+import Text from '../../components/Text';
+import GalleryItem from './components/GalleryItem';
+
+function transformToAssets(individualData) {
+  if (!individualData?.encounters) return [];
+
+  const dataByAsset = transform(
```

Comment:
I don't love transform, because it allows memo to be mutated instead of returned and that often leads to buggy/messy code. Would it be ok to use reduce instead?

Reply:
Switched to `Array.prototype.reduce()` in [4e996b](https://github.com/WildMeOrg/codex-frontend/pull/444/commits/4e996bf7697253627fefe76ce3a6cf495d59879d)